tests/web_api/__init__.py

```diff
@@ -4,6 +4,11 @@
 from openfisca_core.scripts import build_tax_benefit_system
 
 TEST_COUNTRY_PACKAGE_NAME = 'openfisca_country_template'
+
+TEST_EXTENSION_PACKAGE_NAMES = ['openfisca_extension_template']
+
+tax_benefit_system = build_tax_benefit_system(TEST_COUNTRY_PACKAGE_NAME, extensions = TEST_EXTENSION_PACKAGE_NAMES, reforms = None)
+
 distribution = pkg_resources.get_distribution(TEST_COUNTRY_PACKAGE_NAME)
```

Comment:
> Get distribution before using the package to build the tax_benefit_system?

Reply:
Why ? Distribution is only used in `test_headers.py'

Reply:
For debug reasons only. `distribution` is an information about `TEST_COUNTRY_PACKAGE_NAME` that might be useful before building the `tax_benefit_system`.